src/panels/logbook/ha-logbook.ts

```diff
@@ -150,7 +151,9 @@ class HaLogbook extends LitElement {
                 html`
                   <state-badge
                     .hass=${this.hass}
-                    .overrideIcon=${item.icon}
+                    .overrideIcon=${item.icon || (item.domain && !stateObj)
+                      ? domainIcon(item.domain!)
```

Comment:
why don't we want to use `item.icon`?

Reply:
The Parenthesis are bad... good catch.

It should look at item.icon and if no item icon then the other logic